components/features/home/index.tsx

```
@@ -0,0 +1,210 @@
+		wallets,
+		loginButtonText,
+		profile,
+		showModal,
+		setShowModal,
+		login,
+		isLoading,
+		logout,
+	} = useHomeLogin();
+	const { isSmall } = useSizes();
+
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: isLoading

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)